app/assets/javascripts/angular/filters/hyphens.coffee

```diff
@@ -0,0 +1,4 @@
+angular.module "lean-coffee"
+  .filter 'hyphens', ($sce) ->
+    (str) ->
+      str.replace(/\s+/g, '-')
```

Comment:
Are you using SublimeText @ohenrik? If so, adding the following to your user preferences will automatically add a newline at the end of the file:

```json
"ensure_newline_at_eof_on_save": true
```